frontend/js/src/settings/brainzplayer/BrainzPlayerSettings.tsx

```diff
@@ -136,8 +130,9 @@ function BrainzPlayerSettings() {
         />
         <br />
         <small>
-          Spotify requires a premium account to play full songs on third-party
-          websites. To sign in, please go to the{" "}
+          Spotify requires a premium account.
+        <br />
```

Comment:
Not an actual error, but a formatting issue here and other places below, it should be aligned with the text here, same for other `<br />` tags below.
I think installing a code editor would help for stuff like that, as it would fix it for you automatically. If you would like, let me know and I'll help you get set up with it.
```suggestion
<br />
```

Reply:
I fixed this , soz